west.yml

```diff
@@ -129,6 +129,13 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: sdk-homekit
+      path: homekit
+      revision: pull/17/head
+      groups:
+      - homekit
```

Comment:
We need to come up with more groups here, and we need to get it right so that we don't have to update the doc for all homekit users all the time.

I would suggest at the very least:
```
- private
- homekit
```

Reply:
Added `private` group.

Reply:
> and we need to get it right so that we don't have to update the doc for all homekit users all the time.

@carlescufi can you expand on what "right" means?

I guess you are saying that if the user enables `private`, `homekit`, or both of these groups, then you want them to get this repository, `sdk-homekit`. If we add another unrelated repository `foo` which has groups `private`, they'll get both `foo` and `sdk-homekit`. If that's what you want, no objections.

Reply:
> If that's what you want, no objections.

Although I'm kind of confused about why `private` cannot be added later, in that case.

Reply:
Removed `private` group. From HomeKit perspective we can adjust our documentation etc for such generic group later.

Reply:
> I guess you are saying that if the user enables `private`, `homekit`, or both of these groups, then you want them to get this repository, `sdk-homekit`. If we add another unrelated repository `foo` which has group `private`, they'll get both `foo` and `sdk-homekit`. If that's what you want, no objections.

@mbolivar-nordic @ktaborowski @jdstapes 
Yes, that is exactly what I meant, sorry for the confusion, I didn't explain myself well.
What I meant by "get it right" is "make sure we want it and choose the right name". We don't want our users to start scripting away under the premise that `private` will get them all the private repos and then suddenly reverse course by either renaming the group or removing it altogether.